preprocess.py

```diff
@@ -1,28 +1,31 @@
 import numpy as np
-import scipy.signal  as sgn
+import scipy.signal as sgn
+import pdb
 
 
-reduced_leads = ['DI', 'DII', 'V1', 'V2', 'V3', 'V4', 'V5', 'V6']
-all_leads = ['DI', 'DII', 'DIII', 'AVR', 'AVL', 'AVF', 'V1', 'V2', 'V3', 'V4', 'V5', 'V6']
```

Comment:
This change here could be problematic for backward compatibility (older models use the previous order).

Can you remove it and keep the original definition of `reduced_leads`+`all_leads`?